.github/workflows/mlops.yml

```
@@ -0,0 +1,166 @@
+
+      - name: Set up Python
+        run: uv python install
+
+      - name: Sync plugin-eval dependencies
+        run: uv sync --all-extras
+
+      - name: Run pytest
+        run: uv run pytest -q
+
+  model-release:
+    name: Release model to HF (${{ env.HF_NAMESPACE }})
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)